pjlib/src/pj/ioqueue_epoll.c

```diff
@@ -819,6 +819,24 @@ PJ_DEF(int) pj_ioqueue_poll( pj_ioqueue_t *ioqueue, const pj_time_val *timeout)
 	    }
 	}
 
+#if USE_EPOLLONESHOT
```

Comment:
Thanks for the patch and also to @wahjrzd for the report.
But there may be a scenario when the epoll fails to be rearmed, i.e. as written in the first case of the note, when we do not wish to process a reported event (event_cnt < count). That's why initially the rearm was done outside the `event_cnt` loop (which unfortunately, caused the reported bug).

I haven't had any idea on how to fix it though.

Reply:
I had a chance to look at this again, and I think the fix looks good. But we also need to rearm the epoll in the scenario that we're not processing the event as mentioned above. Here's the additional patch (to be applied on top of this one):
[epoll_patch.txt](https://github.com/pjsip/pjproject/files/9036411/epoll_patch.txt)

Reply:
Already merge the patch ,  exclude closing fd.